tool/release.sh

```diff
@@ -33,10 +33,11 @@ fi
 
     rm repc.zip
     rm -rf pkg
-    wasm-pack build --profiling -t web -- --no-default-features
+    wasm-pack build --debug -t web -- --no-default-features
     brotli -f pkg/replicache_client.js pkg/replicache_client_bg.wasm
     mv pkg debug
     wasm-pack build --release -t web -- --no-default-features
+    wasm-opt -O4 -o pkg/replicache_client_bg.wasm pkg/replicache_client_bg.wasm
```

Comment:
I thought that wasm-pack invoked wasm-opt already.

Reply:
I read that somewhere too, but running this takes off like 60KB.